FeedStoreChallenge/Core Data Store/ManagedFeedCache.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ManagedFeedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Hitender Kumar on 22/05/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedCache)
+internal class ManagedFeedCache: NSManagedObject {
```

Comment:
`internal` is the default access control level, so you can remove all the internal keywords to make the code more concise.

`ManagedFeedCache` is not supposed to be inherited thus it should be `final`

Please make it `final`.

```suggestion
final class ManagedFeedCache: NSManagedObject {
```

Reply:
done